### DISCUSSION TITLE
Nvidia Jetpack dependencies with k3os

### DISCUSSION THREAD

**Discussion by Shaked** [rancher/k3os#820] [Ideas]
Hey, 

This might be more relevant under "Discussions", so apologize if it doesn't fit as an issue. 

We are working k3s within our stack, especially in our IoT devices - Nvidia Jetson Xavier. 

We are thinking a lot about maintaining updates and upgrades "over the air" and therefore have been looking into k3os. 

However, Jetson devices are usually installed and updated with [Nvidia's JetPack](https://developer.nvidia.com/embedded/jetpack) (installing CUDA, CUDNN, TRT etc) and we are curious how this can be integrated with k3os if possible. 

Any idea how to approach this? 

Thank you!
Shaked

**Comment by hungnt1** [comment]
@Shaked Have you solved this error yet?

**Comment by dweomer** [comment]
@Shaked I haven't tried but something like this shouldn't be impossible (such might be cost-prohibitive in time however ...)

**Reply by Shaked** [reply]
I haven’t solved it yet. Not sure how to approach it though/where to start CC @hungnt1
